lib/form.js

```diff
@@ -160,12 +160,25 @@ _.extend(Form.prototype, {
     saveValues: function (req, res, callback) {
         callback();
     },
-    getNextStep: function (req) {
+    getNextStep: function (req, res) {
         var next = this.options.next || req.path;
         if (req.baseUrl !== '/') {
             next = req.baseUrl + next;
         }
-        return next;
+        var forks = this.options.forks || [];
+
+        // If a fork condition is met, its target supercedes the next property
+        return _.reduce(forks, function forkFold(result, value) {
+            var evalCondition = function evalCondition(condition) {
```

Comment:
No, good point - this is how it ended up in hof-controllers but the logic is simpler now - appending /edit and redirecting to /confirm still live within hof.

I'll tidy this